src/locales/en.json

```
@@ -77,7 +77,7 @@
     "group": "Group selection",
     "ungroup": "Ungroup selection",
     "collaborators": "Collaborators",
-    "gridMode": "Grid mode",
+    "gridMode": "Show grid",
```

```suggestion
    "showGrid": "Show grid",
```

to keep it consistent

---

Copy can generally be divorced from code variable naming.

---

True.. but while we at it.. if we were about to introduce it today it would be `showGrid` :) and less confusing for new comers/